File: businesslib/src/main/java/com/wujia/businesslib/event/EventMemberChange.java
package com.wujia.businesslib.event;

import org.greenrobot.eventbus.Subscribe;
import org.greenrobot.eventbus.ThreadMode;

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-31
 * description ：
 */
public class EventMemberChange extends AbsEventEntiry<EventMemberChange> {

    public EventMemberChange() {

    }

    public EventMemberChange(IMiessageInvoke<EventMemberChange> invoke) {
        super(invoke);
    }

    @Subscribe(threadMode = ThreadMode.MAIN)
    public void onEventBus(EventMemberChange event) {
        invoke(event);
    }
}

File: businesslib/src/main/java/com/wujia/businesslib/event/EventMemberChange.kt
package com.wujia.businesslib.event

import org.greenrobot.eventbus.Subscribe
import org.greenrobot.eventbus.ThreadMode

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-31
 * description ：
 */
class EventMemberChange : AbsEventEntiry<EventMemberChange> {

    constructor() {

    }

    constructor(invoke: IMiessageInvoke<EventMemberChange>) : super(invoke) {}

    @Subscribe(threadMode = ThreadMode.MAIN)
    override fun onEventBus(event: EventMemberChange) {
        invoke(event)
    }
}
